libs/SmartStore/src/com/salesforce/androidsdk/smartstore/store/SmartStore.java

```diff
@@ -1647,6 +1647,24 @@ public String getSQLCipherVersion() {
 		return TextUtils.join(" ", queryPragma("cipher_version"));
 	}
 
+	/**
+	 * Get SQLCipher provider version
+	 *
+	 * @return SQLCipher provider version
+	 */
+	public String getCipherProviderVersion() {
+		return TextUtils.join(" ", queryPragma("cipher_provider_version"));
+	}
+
+	/**
+	 * Get SQLCipher FIPS status
+	 *
+	 * @return "0" when using the community edition or the commercial edition and "1" when using the FIPS enabled enterprise edition
+	 */
+	public String getCipherFIPSStatus() {
```

Comment:
NIT:  Feels like this might be a bit more user friendly as a boolean instead of `"0"` or `"1"` strings.